### PR TITLE
Enhancement/add slash commands

### PR DESCRIPTION
**Pull request by bmw2621**
add slash commands

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 from dotenv import load_dotenv
 import os
-import discord
 import datetime
 from pg2db import *
+import discord
```
Comment (rbrady) [inline main.py:5]: There is a specific order your imports should be in.  You start with standard library, followed by third party libraries and finally your local application imports.  All of the groups should be separated by a blank line.


Please see https://www.python.org/dev/peps/pep-0008/#imports

**Review by rbrady — CHANGES_REQUESTED** [review]
You don't want to check in the __pycache_ dir or any .pyc files.  Please update your .gitignore and remove this files from the PR.

Other comments for outside this PR:

The main.py file could use a run through with flake8 to show many of the style issues that do not comply with PEP8.

There's a couple of really long numerical identifiers used in the script that should really be commented and/or assigned to a well named variable.

We typically don't try to execute logic in a module without a very specific reason.  The format for a python script is to have a main() function and then at the bottom of the module have the following code:

if __name__=="__main__":
    main()

**Comment by bmw2621** [comment]
@rbrady  Im not sure the flake8 formatting made it much better.  Is there a Prettier type autoformatting solution that is industry standard?

**Comment by rbrady** [comment]
It’s looking a lot better.  Flake8 is a linter, it will find what parts of your code do not fit pep8 standards.  Black is a formatter that will help you with white space and some common style issues, but they do two different things.  They work well together.

the only thing that looks super weird to me now is defining a function under the if name == main statement.

**Comment by bmw2621** [comment]
I couldnt get black working in VSCode, so I opened the project in PyCharm.  This is why I should just use PyCharm.

**Pull request merged**